.github/workflows/test.yml

```diff
@@ -57,6 +57,13 @@ jobs:
         with:
           command: test
           args: --locked
+  shellcheck:
+    runs-on: ubuntu-22.04
```

Comment:
It is odd that GitHub actions offers `ubuntu-22.04`, which _does_ have a version of `shellcheck` with the `-o all` option, but that VM is _not_ the same as `ubuntu-latest`...which is still 20.04 a/o this writing.

Not critical, but perhaps a comment here to explain this would help future readers of this file who may notice the discrepancy between the image used on this job compared to the image used on the other jobs.